### DISCUSSION TITLE
Translation into Russian 🇷🇺 need to work

### DISCUSSION THREAD

**Discussion by Dumitru88888** [xushengfeng/eSearch#244] [General]
Translation into Russian 🇷🇺 I need to work on a very noticeable Chinese language

**Comment by xushengfeng** [comment]
https://github.com/xushengfeng/eSearch/blob/master/lib/translate/readme.md

**Comment by xushengfeng** [comment]
now tool can output Chinese with English，You don't need to be proficient in Chinese to translate

**Comment by Dumitru88888** [comment]
Not all words have been translated into Russian

  
>Суббота, 27 июля 2024, 13:07 +03:00 от xushengfeng ***@***.***>:
> 
> 
>now tool can output Chinese with English，You don't need to be proficient in Chinese to translate
>—
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>You are receiving this because you authored the thread. Message ID:  <xushengfeng/eSearch/repo-discussions/244/comments/10166677 @ github . com> 
 
 
С уважением,
Dima 1998
***@***.***
